### ISSUE TITLE
The industry has a problem. There is no way to see new NFTs.

### ISSUE THREAD

**Issue by nickjuntilla** [MetaMask/metamask-extension#21597]
### What is this about?

The blockchain is supposed to be an open industry anyone can participate in right?

We have an NFT rewards program and customers claim NFTs from our software. We have the option to send that NFT to their metamask. Right now all of those NFTs end up hidden and the process to unhide them is difficult in the mobile app and impossible in the browser extension. This makes our project and any new project using NFTs very difficult to manage. We can't expect all our users to mint the NFTs themselves. My questions are.

* Is there some kind of safe list?
* What are the methods you use to decide what NFTs to allow to be visible?

We understand there are scams and malicious actors in the NFT community, but we also need to grow as an industry and if the policy to to prevent the display of all NFTs made after a certain date that is not going to make it very easy to grow this industry. A better policy might be to allow all NFTs and move some to the bottom of the list. Users could add their favorite NFTs to a priority list or there could be a "Show more" button.  The current method of "block all and ask questions later" although may be a temporary solution to scam complaints it is a huge problem for NFT projects like ours.

We appreciate your help and guidance on this issue! Thank you!

### Scenario

-User comes to our website and claims a rewards NFT
-They enter their metamask wallet address
-Our system sends them an NFT
-The NFT is sent and the transaction is confirmed on the blockchain
-The user cannot see the NFT and there is no way for them to see the NFT

### Acceptance Criteria

-User comes to our website and claims a rewards NFT
-They enter their metamask wallet address
-Our system sends them an NFT
-The NFT is sent and the transaction is confirmed on the blockchain
-The NFT shows up, perhaps in "Received" or "Quarantined" section, but the user has access to it.

**Comment by alfeng6** [comment]
Hi! NFT Product Manager for MetaMask here. Thank you for message — I agree that there should be some solution to minimize the number of false positives and false negatives when it comes to detecting the right tokens and NFTs. That's something high on our priority list.

In the meantime, I would recommend checking out the watchAsset function found here: https://docs.metamask.io/wallet/reference/rpc-api/

The tldr is that it can directly prompt your users to add an asset to their wallet.
